rebuild-srpms.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh -eu
+
+TMPDIR=$(mktemp -d --suffix "-rebuild-srpms")
+trap 'rm -rf "$TMPDIR"' EXIT
+
+rebuild_srpm() {
+  local tmpdir=$1 srpm=$2 arch=$3
+  local topdir_tmp=$(exec mktemp -p "$tmpdir" -d --suffix "-$srpm.$arch")
+
+  echo "Rebuilding $srpm for $arch"
+
+  dist=$(rpm -qp --qf "%{release}\n" "$srpm" | grep -Po "\.fc\d+")
```

Comment:
this is ugly, but I don't know any other way apart from extracting SPEC, trying to parse it with different dist tag and find difference which is (however) is unreliable